server/sonar-web/src/main/js/apps/system/item-log-level.js

```diff
@@ -23,17 +23,15 @@ import { translate } from '../../helpers/l10n';
 
 const LOG_LEVELS = ['INFO', 'DEBUG', 'TRACE'];
 
-export default React.createClass({
-  getInitialState() {
-    return { level: this.props.value };
-  },
+export default class ItemLovLevel extends React.PureComponent {
+  state = { level: this.props.value };
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the use of "this". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS2990)